cmake/pcl_utils.cmake

```diff
@@ -441,3 +441,24 @@ function(find_python_module module)
   find_package_handle_standard_args(PY_${module} DEFAULT_MSG PY_${module_upper})
 endfunction(find_python_module)
 
+
+###############################################################################
+# Checks if the current generator is an IDE
+# _out The boolean result
+macro(check_if_ide _out)
+  # Current known pool of generators
+  set(_ides
+    "Xcode"
+    "Visual Studio 14 2015"
+    "Visual Studio 15 2017"
```

Comment:
Is there a reason to have specific versions here? I'd prefer a future-proof check if possible.

Reply:
No. It was a literal copy paste from the supported generators list. I'll change it to `"Visual Studio*"`.